### PR TITLE
EXLM-1130 Fixed TOC product logo and theme style

### PR DESCRIPTION
**Pull request by ruhisingh1**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: EXLM-1130

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/en/docs/creative-cloud-enterprise-learn/cce-learning-hub/overview
- After: https://bugfix-exlm-1130--exlm--adobe-experience-league.hlx.live/en/docs/creative-cloud-enterprise-learn/cce-learning-hub/overview

### CODE REVIEW

File: blocks/toc/toc-solutions.css [hunk 1]
```diff
@@ -541,4 +541,19 @@
     padding-bottom: 3px;
     margin-right: 3px;
     color: var(--exlm-toc-solution-6);
+}
+.toc.CircleExperienceCloud ul#product li>.product-icon {
+    background: url("../../icons/solutions/CircleExperienceCloud.svg") no-repeat center center;
```
Comment (noormd08) [inline blocks/toc/toc-solutions.css:546]: rename the svg file to experience-cloud.svg

File: blocks/toc/toc-solutions.css [hunk 2]
```diff
@@ -541,4 +541,19 @@
...
+}
+.toc.CircleExperienceCloud ul li a:not(a[href^="#"]).is-active {
+    padding-left: 10px;
+    border-left-color: var(--exlm-toc-solution-2);
+}
+.toc.CircleExperienceCloud ul>li>p>a[href^="#"].js-toggle.is-open:before {
```
Comment (noormd08) [inline blocks/toc/toc-solutions.css:552]: Can we rename this class and avoid multiple chaining ?

File: blocks/toc/toc-solutions.css (hunk 1 above)
Comment (ruhisingh1) [reply blocks/toc/toc-solutions.css:546]: The svg file names are mapped with the solution name. So, we cannot change it.

File: blocks/toc/toc-solutions.css (hunk 2 above)
Comment (ruhisingh1) [reply blocks/toc/toc-solutions.css:552]: The class names are mapped with the solution name. So, we cannot change it. 
Regarding multiple chaining, the same code has been added for all other solution classes.

**Review by noormd08 — APPROVED** [review]

**Pull request merged**